app/grants/templates/grants/detail/index.html

```diff
@@ -69,6 +81,57 @@
       </div>
     </div>
+    <div class="modal fade" id="startVerification" tabindex="-1" role="dialog" aria-labelledby="startVerification" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content text-center">
+          <div class="modal-header" style="border-bottom: none">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body px-5">
+            <img height="45" class="mb-4" src="{% static "v2/images/badge-verify.svg" %}" >
+            <h5 class="font-weight-bold mb-3">Verify Grant Ownership</h5>
+            <p class="mb-4">Verify your grant ownership to ensure that your supporters are contributing to the correct grant.</p>
+            <button id="triggerTwitter" class="btn btn-lg btn-gc-blue" data-toggle="modal" data-target="#startTwitterVerification">Verify with twitter</button>
+          </div>
+          <div class="modal-footer" style="border-top: none">
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="modal fade" id="startTwitterVerification" tabindex="-1" role="dialog" aria-labelledby="startTwitterVerification" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content text-center">
+          <div class="modal-header" style="border-bottom: none">
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body px-5">
+            <img height="45" class="mb-4" src="{% static "v2/images/badge-verify.svg" %}" >
+            <h5 class="font-weight-bold mb-3">Verify Grant Ownership with Twitter</h5>
+            <p class="mb-4 text-left"><b>Step 1:</b> Send a tweet from your account (<b>{{ grant.twitter_handle_1 }}</b>) saying <b>
+              I am verifying my ownership of the {{ grant.title }} {{ user_code }}.</b> <br>
+              <a class="twitter-share-button"
+                href="https://twitter.com/intent/tweet"
+                data-size="small"
+                data-text="I am verifying my ownership of the {{ grant.title }}. \n{{ user_code }}\n"
+                data-url="https://gitcoin.co{{ grant.get_absolute_url }}"
+                data-hashtags="grants,round7">
+              Tweet
+              </a>
+            </p>
+            <p class="mb-4 text-left"><b>Step 2:</b>Click the "Verify" button below</p>
+            <button class="btn btn-lg btn-gc-blue mb-4" id="twitterVerification">Verify</button>
+            <span id="validation-errors"></span>
+
+          </div>
+          <div class="modal-footer" style="border-top: none">
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
^ could we reduce the overall size of this to make it look neater ? It looks very big on the screenshot